src/views/AnnualReport.vue

```diff
@@ -888,9 +901,14 @@ export default {
           }
           filing = res.data.filing
           this.haveChanges = false
-        }).catch(error => {
+        }).catch(async error => {
           if (error && error.response && error.response.status === PAYMENT_REQUIRED) {
-            this.paymentErrorDialog = true
+            const errCode = this.getErrorCode(error)
+            if (errCode) {
+              this.bcolObj = await this.getErrorObj(errCode.payment_error_type)
+            } else {
+              this.paymentErrorDialog = true
```

Comment:
In `resetErrors()`, should clear this.bcolObj and this.paymentErrorDialog.

Same in other 2 filings.

Reply:
whoops

Reply:
Fixed.